store/conf/attrs/zimbra-attrs.xml

```diff
@@ -5260,7 +5260,7 @@ TODO: delete them permanently from here
   <desc>enable/disable MAPI (Microsoft Outlook) Connector</desc>
 </attr>
 
-<attr id="1128" name="zimbraDumpsterEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountInfo,accountInherited,domainAdminModifiable,domainInherited" since="7.0.0">
+<attr id="1128" name="zimbraDumpsterEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,account,cos,domain" flags="accountInfo,accountCosDomainInherited,domainAdminModifiable,domainInherited" since="7.0.0">
```

Comment:
check how this works in upgrade scenario

Reply:
We don't require any extra work to do, this attribute will be reflected while upgradation.

Reply:
Will new `<globalConfigValue>FALSE</globalConfigValue>` , which is added on https://github.com/Zimbra/zm-mailbox/pull/1314, be applied on an existing attribute at upgrade?

Reply:
@k-kato Verified this on after upgrading the build, we are seeing the expected value :
`zimbra@perf-zcs-4-zimbradev-com:~$ zmprov gcf zimbraDumpsterEnabled`
`zimbraDumpsterEnabled: TRUE`
`zimbra@perf-zcs-4-zimbradev-com:~$ zmprov gcf zimbraDumpsterUserVisibleAge`
`zimbraDumpsterUserVisibleAge: 30d`
`zimbra@perf-zcs-4-zimbradev-com:~$ zmprov gcf zimbraDumpsterPurgeEnabled`
`zimbraDumpsterPurgeEnabled: TRUE`

Reply:
@zimsuchitgupta thank you for the confirmation.